app/src/main/java/com/android/sample/ui/activity/EditActivity.kt

```diff
@@ -526,7 +526,7 @@ fun EditActivityScreen(
             Button(
                 enabled = title.isNotEmpty() && description.isNotEmpty(),
                 onClick = {
-                  if (hourDateViewModel.isBeginGreaterThanEnd(
```

Comment:
checks if start time is before the end time. (if not an error toast will be displayed in the screen with the message: "The start time must be before the end time.".